dracut/repartition/endless-repartition.sh

```diff
@@ -144,6 +144,12 @@ if [ -n "$swap_start" ]; then
 start=$swap_start, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F"
 fi
 
+# Randomize PARTUUIDs
```

Comment:
You can pass multiple -e options to sed. So please combine these 2 with the sed that deletes last-lba. That way we run 1 sed process instead of 3.